test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/ProjectDirectory.cs

```diff
@@ -112,6 +113,14 @@ void CreateDirectoryTargets(string projectRoot)
 ";
                 File.WriteAllText(Path.Combine(projectRoot, "Directory.Build.targets"), text);
             }
+
+            void CopyGlobalJson(string solutionRoot, string projectRoot)
+            {
+                var srcGlobalJson = Path.Combine(solutionRoot, "global.json");
+                var destinationGlobalJson = Path.Combine(projectRoot, "global.json");
+
+                File.Copy(srcGlobalJson, destinationGlobalJson);
```

Comment:
The global.json is only created when running `./build.cmd` so this will blow up if you're doing `dotnet test` prior to doing a `./build`. I'd do a file exists check

Reply:
I can add a check. But the scenario is kinda broken regardless. Without the right sdk downloaded \ being pinned, there are no guarantees builds would succeed.